org.envirocar.app/res/color/dashboard_indicator_selector.xml

```diff
@@ -1,25 +1,19 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!--
-
     Copyright (C) 2013 - 2019 the enviroCar community
-
     This file is part of the enviroCar app.
-
     The enviroCar app is free software: you can redistribute it and/or
     modify it under the terms of the GNU General Public License as published
     by the Free Software Foundation, either version 3 of the License, or
     (at your option) any later version.
-
     The enviroCar app is distributed in the hope that it will be useful, but
     WITHOUT ANY WARRANTY; without even the implied warranty of
     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
     Public License for more details.
-
     You should have received a copy of the GNU General Public License along
     with the enviroCar app. If not, see http://www.gnu.org/licenses/.
-
 -->
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
-    <item android:color="@color/cario_color_primary_dark" android:state_enabled="false" />
-    <item android:color="@color/cario_warning_red" android:state_enabled="true" />
+    <item android:color="@color/cario_color_primary_dark" android:state_activated="false" />
+    <item android:color="@color/cario_warning_red" android:state_activated="true" />
 </selector>
```

Comment:
I think it is a little bit misleading, to set the primary color for `state_activated="false"`, since the primary color usually indicates an active state. Of course, it will be handled properly in the code by setting `this.bluetoothIndicator.setActivated(!event.isBluetoothEnabled);`. But why not setting the state as active when Bluetooth is enabled? That would be more understandable.

Reply:
This is how it was implemented previously, `state_enabled="false"` is used for primary colour, I totally agree with you sir, true should be used when active. I will change the implementation now.